AwsEncryptionSDK/runtimes/go/ImplementationFromDafny-go/go.mod

```diff
@@ -7,30 +7,30 @@ require github.com/aws/aws-cryptographic-material-providers-library/releases/go/
 replace github.com/aws/aws-cryptographic-material-providers-library/releases/go/mpl v0.0.0 => ../../../../../aws-cryptographic-material-providers-library/AwsCryptographicMaterialProviders/runtimes/go/ImplementationFromDafny-go/
 
 require (
-	github.com/aws/aws-cryptographic-material-providers-library/releases/go/dynamodb v0.0.0
-	github.com/aws/aws-cryptographic-material-providers-library/releases/go/kms v0.0.0
-	github.com/aws/aws-cryptographic-material-providers-library/releases/go/mpl v0.0.0
-	github.com/aws/aws-cryptographic-material-providers-library/releases/go/primitives v0.0.0
+	github.com/aws/aws-cryptographic-material-providers-library/releases/go/dynamodb v0.0.3
```

Comment:
Isn't our process replacing this package with the local version in `runtime/go`. But put the actual version with no replacement in `releases/go` directory?

Reply:
We do have a `replace` in place.

Reply:
Also made changes to do a version agnostic replace.

Reply:
Non blocking but I would rather put v0.0.0 (a version thats not released) in require to signal replace directives is used in the go mod.

Reply:
Not really - I'd rather have the version number and do a replace. The issue with v0.0.0 is if you really want to test against a version you have to explicitly specify it and then go mod and stuff. With the right version in place, you can just comment out the replace directive and it will work.